packages/cli/src/commands/destroy/cell/__tests__/cell.test.js

```diff
@@ -17,27 +23,29 @@ jest.mock('@redwoodjs/structure', () => {
 })
 
 import fs from 'fs-extra'
+import { vol } from 'memfs'
+import { vi, beforeEach, afterEach, test, expect } from 'vitest'
 
 import '../../../../lib/test'
 
 import { files } from '../../../generate/cell/cell'
 import { tasks } from '../cell'
 
 beforeEach(() => {
-  jest.spyOn(console, 'info').mockImplementation(() => {})
-  jest.spyOn(console, 'log').mockImplementation(() => {})
+  vi.spyOn(console, 'info').mockImplementation(() => {})
+  vi.spyOn(console, 'log').mockImplementation(() => {})
 })
 
 afterEach(() => {
-  fs.__setMockFiles({})
-  jest.spyOn(fs, 'unlinkSync').mockClear()
+  vol.reset()
+  vi.spyOn(fs, 'unlinkSync').mockClear()
```

Comment:
Is this still needed with memfs?

Reply:
In this particular case the tests still pass without this line. However, if you log the content of the fs at the start of each test you can see content being carried over between these two test cases. 

I think in general this would still be required depending on the assumptions made within tests. I'd still probably recommend adding a reset after each test if that is consistent with the testing logic

Reply:
Yeah, not questioning `vol.reset()` at all 🙂 
I was wondering about `vi.spyOn(fs, 'unlinkSync').mockClear()`. `fs` is the memfs mock here, right? What does calling `mockClear()` on memfs.unlinkSync even do?